scripts/merge-conflicts.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -euo pipefail
+
+USERNAME="$1"
+EMAIL="$2"
+
+orginal_url="https://github.com/ethereum/solidity.git"
+git remote add english "$orginal_url"
+
+git config user.name "$USERNAME"
+git config user.email "$EMAIL"
+
+## Fetch from translated origin
+git fetch english --quiet
+sync_branch="sync-$(git describe --tags --always english/develop)"
+
+# pass the hash and the branch name to action "create PR"
+echo "::set-output name=branch_name::$sync_branch"
+
+# check if sync branch exists
+if (git ls-remote --exit-code --heads origin "$sync_branch") 
```

Comment:
I don't think a subshell is really needed here.
```suggestion
if git ls-remote --exit-code --heads origin "$sync_branch"
```

Reply:
Done in #23.